jenkins/jobs/integration_tests.pipeline

```diff
@@ -9,7 +9,7 @@ def TIMEOUT = 7200
 KEEP_TEST_ENV=(env.KEEP_TEST_ENV)
 
 if (env.TESTS_FOR) {
-  if ( (env.TESTS_FOR).startsWith('feature_tests') || (env.TESTS_FOR).startsWith('e2e_tests') ) {
+  if ( (env.TESTS_FOR).startsWith('feature_tests') || ( (env.TESTS_FOR).startsWith('e2e_tests') && !(env.GINKGO_FOCUS).equals('integration'))) {
```

Comment:
/hold
@huutomerkki We might need basic here as well

Reply:
Added it as well